src/openrct2/ride/TrackDesignRepository.cpp

```diff
@@ -33,7 +33,7 @@
 
 extern "C"
 {
-    #include "track_design.h"
+    #include "TrackDesign.h"
```

Comment:
As this now a C++ header file, you should probably remove the `extern C` block.

Reply:
It is not _really_ a C++ header file, though. I didn't do anything else than renaming it.

Reply:
It is still used by other C files so you can't remove it.